File: myblog-server/src/main/java/com/site/blog/service/impl/LinkServiceImpl.java
package com.site.blog.service.impl;

import com.site.blog.model.entity.Link;
import com.site.blog.mapper.LinkMapper;
import com.site.blog.service.LinkService;
import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 友情链接表 服务实现类
 * </p>
 *
 * @author: 南街
 * @since 2019-09-02
 */
@Service
public class LinkServiceImpl extends ServiceImpl<LinkMapper, Link> implements LinkService {

}

File: myblog-kt/src/main/kotlin/com/site/blog/service/impl/LinkServiceImpl.kt
package com.site.blog.service.impl

import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl
import com.site.blog.mapper.LinkMapper
import com.site.blog.model.entity.Link
import com.site.blog.service.LinkService
import org.springframework.stereotype.Service

/**
 *
 *
 * 友情链接表 服务实现类
 *
 *
 */
@Service
class LinkServiceImpl : ServiceImpl<LinkMapper?, Link?>(), LinkService